docker/bench_py27_cython.Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Falcon Framework Maintainers
 RUN pip install cython
 
 # We don't currently benchmark JSON deserialization, but in the future we might
-RUN pip install ujson
+# RUN pip install ujson
```

Comment:
You may want to change this to orjson to mirror docker/bench_py37_cython.Dockerfile

Reply:
No, I may not I think. `orjson` does not support Python 2.7.